includes/admin/meta-boxes/class-wc-meta-box-product-images.php

```diff
@@ -50,11 +74,18 @@ public static function output( $post ) {
 							continue;
 						}
 
+						// Allow for extra info or action to be exposed for this attachment, but restrict it's output.
+						$attachment_info_action = wp_kses(
+							apply_filters( 'woocommerce_product_gallery_item_info_action', '', $thepostid, $attachment_id ),
```

Comment:
In this case is best to use `do_action()` where you want to output the content, also we don't need to be so strict here with sanitization, since who is going to implement this action should be already validate and sanitize it.

Also this action name could be changed to `woocommerce_admin_after_product_gallery_item`.

Reply:
Thank you for the feedback, @claudiosanches!
I made the changes you suggested. Please review when possible.